pyvista/core/filters/image_data.py

```diff
@@ -1226,6 +1258,12 @@ def contour_labels(  # type: ignore[misc]
         using ``select_inputs`` converts previously-internal boundaries into external
         ones.
 
+        Disable smoothing to generate staircase-like surface. Without smoothing, the
```

Comment:
It would be easier to understand if there is also an example of when the smoothing parameters are changed.

Reply:
An example with `smoothing_scale` is added in [17aa6fa](https://github.com/pyvista/pyvista/pull/7021/commits/17aa6fa9a1c4f3d21ebac2071ad7ade3f468b856).

Reply:
For the example with `smoothing_scale=0.5`, this is what the final docs will look like (with `closed_surface=True`)

![](https://raw.githubusercontent.com/pyvista/pyvista/f12b9ab2795d710c783f7d443cb5bed5422faf3a/tests/doc/doc_image_cache/pyvista-ImageDataFilters-contour_labels-1_06_00.jpg)

Compared to the default with `smoothing_scale=1.0`


![](https://raw.githubusercontent.com/pyvista/pyvista/f12b9ab2795d710c783f7d443cb5bed5422faf3a/tests/doc/doc_image_cache/pyvista-ImageDataFilters-contour_labels-1_01_00.jpg)

These images require `closed_surface=True` from #7028 which is not yet merged as an option, so the examples as they currently are in this PR don't yet appear like this.